HTMLCSSJS/homework.html

```diff
@@ -0,0 +1,93 @@
+<html lang="en">
+  <head>
+    <link rel="stylesheet", href="homework2.css" />
```

Comment:
- The file `homework2.css` does not found in your repository. 
- No need for the comma between two HTML attributes